datanode/gateway/graphql/schema.graphql

```diff
@@ -845,6 +842,21 @@ type Query {
     pagination: Pagination
   ): ReferralSetRefereeConnection!
 
+  "Get the current referral program"
+  currentReferralProgram: CurrentReferralProgram
+
+  "Get the current volume discount program"
+  currentVolumeDiscountProgram: VolumeDiscountProgram
+
+  volumeDiscountStats(
+    "Optional epoch to get statistics from. If not provided, the latest statistics are returned"
+    epoch: Int
+    "Optional party ID"
+    partyId: ID
+    "Optional pagination information"
+    pagination: Pagination
+  ): VolumeDiscountStatsConnection!
```

Comment:
We've been making an effort to keep the API's in alphabetically order so it's easier to look them up in the documentation.